mahiru/policy/rules.py

```diff
@@ -122,13 +122,15 @@ def __init__(
             self,
             data_asset: Union[str, Identifier],
             compute_asset: Union[str, Identifier],
+            output: str,
             collection: Union[str, Identifier]
             ) -> None:
         """Create a ResultOfIn rule.
 
         Args:
             data_asset: The source data asset.
             compute_asset: The compute asset used to process the data.
+            output: The workflow output that produced the result Asset.
```

Comment:
So what you mean with 'output' refers to sort of the variable name of the workflow output value that is then feed into the result Asset? It's a bit implicit from the docstring alone, this is what I figured from the examples.

Reply:
Yes, it is the name of the output of the workflow step that produced the asset that's going to be in the collection. I've tried to clarify that a bit, how's this?

Reply:
Yes, looks good! :)